components/developer/gdb/patches/gdb.gdb.sol-thread.c.patch

```diff
@@ -1,27 +1,34 @@
---- gdb-7.10.1/gdb/sol-thread.c.~1~	Thu Feb 19 03:58:07 2015
-+++ gdb-7.10.1/gdb/sol-thread.c	Fri Feb 26 23:04:44 2016
-@@ -246,16 +246,18 @@
+--- ./gdb-7.10.1/gdb/sol-thread.c.~1~	Thu Feb 19 03:58:07 2015
```

Comment:
the patch is incorrect:
/usr/gnu/bin/patch -d /export/home/alp/srcs/tests/oi-userland/components/developer/gdb/gdb-7.10.1 --strip=1 --backup --version-control=numbered < patches/gdb.gdb.sol-thread.c.patch
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- ./gdb-7.10.1/gdb/sol-thread.c.~1~  Thu Feb 19 03:58:07 2015
|+++ ./gdb-7.10.1/gdb/sol-thread.c      Sat Jul 30 08:27:12 2016